<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminPageDialog</class>
 <widget class="QDialog" name="adminPageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Main Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>My covid record update document page</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="todayDate">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Latest update user data list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Covid-19 test result and Vaccine record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Upload or Modify new data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter user ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userIdLineEdit">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter user name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userNameLineEdit">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="userIDPushButton">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="userNamePushButton">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter user email address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userEmailLineEdit">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter user mobile number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userMobileLineEdit">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="userEmailPushButton">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="userMobilePushButton">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
